src/platform/site-wide/maintenance-banner/config.js

```diff
@@ -0,0 +1,22 @@
+// Node modules.
+import moment from 'moment';
+
+// Derive startsAt and expiresAt.
+const startsAt = moment.utc('2020-06-02T17:00:00.248Z').local();
+const expiresAt = moment.utc('2020-06-03T19:00:00.248Z').local();
+
+// Format startsAt and expiresAt.
+const formattedStartsAt = startsAt.format('dddd M/D, h:mm a');
+const formattedExpiresAt = expiresAt.format('dddd M/D, h:mm a');
+
+export default {
+  id: '1',
```

Comment:
I suppose we have to increment this each time there's new maintenance?

Reply:
Yupp, whenever there's a new downtime message we want to show since this is what is used in localStorage to track if it's been dismissed or not.